docs/en/stack/ml/anomaly-detection/ml-limitations.asciidoc

```diff
@@ -234,6 +234,13 @@ Explorer** nor the **Single Metric Viewer** can plot and display an anomaly
 chart for the job. In these cases, the charts are not visible and an explanatory 
 message is shown.
 
+When the aggregation interval of the {dfeed} and the bucket span of the job
+don't match, the values of the chart plotted in both the **Single Metric
+Viewer** and the **Anomaly Explorer** differ from the actual values of the job.
+To avoid this behavior, make sure that the aggregation interval in the {dfeed}
+configuration and the bucket span in the {anomaly-job} configuration have the
+same values.
```

Comment:
> same values.

I think we need to include a "However, ..." and mention why that isn't the recommendation from the back-end point of view.

Reply:
@lcawl I added the other side of the story via 30d6c79.